mlir/include/QEC/Transforms/Passes.td

```diff
@@ -37,4 +37,14 @@ def CommuteCliffordTPPRPass : Pass<"commute_ppr"> {
     let constructor = "catalyst::createCommuteCliffordTPPRPass()";
 }
 
+def CommuteCliffordPastPPMPass : Pass<"ppr_to_ppm"> {
```

Comment:
I think the _pass_ name was fine as "Absorb..." because the combined action of the commutation and the deletion pattern is that the PPRs are "absorbed" (when possible), but it's not too important 😅